test/confirm-blocklist.test.js

```diff
@@ -0,0 +1,37 @@
+const test = require("tape");
+const { testDomain } = require("./test.util.js");
+const sqlite3 = require("sqlite3").verbose();
+const db = new sqlite3.Database("./tranco.db");
+
+function runTests(config) {
+  test("check config blocklist against top Tranco domains", async (t) => {
+    let foundWhitelistedDomains = "";
```

Comment:
Replace `foundWhitelistedDomains` from type `string` to type `array`. 

`let foundWhitelistedDomains = []`

Reply:
done